addons/web/static/src/model/relational_model/relational_model.js

```diff
@@ -660,17 +660,22 @@ export class RelationalModel extends Model {
         }
     }
 
-    async _webReadGroup(config, firstGroupByName, orderBy) {
+    async _webReadGroup(config, orderBy) {
+        const aggregates = Object.values(config.fields).filter(
+            (field) => field.group_operator && field.name in config.activeFields
+        ).map(
+            (field) => `${field.name}:${field.group_operator}`
```

Comment:
why can't this be done server side, based on the list of activeFields?

Reply:
In this case, it is because right now `group_operator` (`_description_group_operator`, make sure this group_operator is possible for this field.) sent to the client is the same as `group_operator` in server side (just a default class value). But it is definitely worth it to clarifying it and changing the naming (maybe => default_aggregate). 

IMO, it is not the job of the server to choose the aggregate to be done on the field (maybe in the future, we may be able to choose different aggregate depending on the views).

Also the private version of _read_group only accepts explicit aggregates and I personally find it much cleaner to read and to debug.

What do you think ?

Reply:
you could add a key `aggregate` in activeField, s.t. if it is set to false, you ignore it. In kanban_controller, in the modelParams getter, you could set `aggregate` to false on every field that isn't a progressbar field.